briefcase/briefcase.go

```diff
@@ -4,10 +4,11 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
-	"github.com/hootsuite/vault-ctrl-tool/v2/metrics"
 	"io/ioutil"
 	"time"
 
+	"github.com/hootsuite/vault-ctrl-tool/v2/metrics"
```

Comment:
_nit._ Imports in this file should probably just be a group for std lib and a group for everything else.